Formula/kubeless.rb

```diff
@@ -0,0 +1,35 @@
+class Kubeless < Formula
+  desc "Kubernetes Native Serverless Framework"
+  homepage "https://github.com/kubeless/kubeless"
+  url "https://github.com/kubeless/kubeless.git",
+    :tag => "v0.2.3",
+    :revision => "bfefb404e22a209659ef06b98a51051d72dd5c21"
+  depends_on "go" => :build
+  depends_on "kubernetes-cli" => :recommended
+
+  def install
+    commit = `git rev-parse --short HEAD`.chomp
```

Comment:
Use `Utils.popen_read("git rev-parse --short HEAD").chomp` instead. It's better for my long-term happiness 🤣.